.github/workflows/pr_integration_tests.yml

```diff
@@ -170,3 +170,14 @@ jobs:
           env_vars: OS,PYTHON
           fail_ci_if_error: true
           verbose: true
+      - name: Test go
```

Comment:
My only concern here is that we are adding a step to an already slow-running test suite which will further slow it down. Would it make sense to run the Go server tests as part of the rest of the test suite so that it can be parallelized?

Reply:
this is far, I'll make sure the Go tests can get parallelized